src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import org.broadinstitute.dsde.rawls.model._
+
+trait RawlsBillingProjectComponent {
+  this: DriverComponent =>
+
+  import driver.api._
+
+  class RawlsBillingProjectTable(tag: Tag) extends Table[RawlsBillingProjectShort](tag, "BILLING_PROJECT") {
+    def projectName = column[String]("projectName", O.PrimaryKey)
```

Comment:
Does it make a big difference?  VARCHAR isn't a standard Slick type, so if it's not clear what the performance improvement would be, I'd prefer to keep it more generic.